src/ReportPortal.Client/Abstractions/Requests/CreateLogItemRequest.cs

```diff
@@ -25,12 +25,22 @@ public class CreateLogItemRequest
         /// Date time of log item.
         /// </summary>
         public DateTime Time { get; set; } = DateTime.UtcNow;
+        
+        /// <summary>
+        /// A level of log item serialized as string to support custom levels.
+        /// </summary>
+        [JsonPropertyName("level")]
+        public string LevelText { get; set; } = "INFO";
 
         /// <summary>
-        /// A level of log item.
+        ///     A typed helper property for programs that prefer the typed LogLevel enum.
         /// </summary>
-        [JsonConverter(typeof(JsonStringEnumConverterEx<LogLevel>))]
-        public LogLevel Level { get; set; } = LogLevel.Info;
+        [JsonIgnore]
+        public LogLevel Level
```

Comment:
To discuss: should we mark it as `[Obsolete]`?

Reply:
Agree, added [Obsolete]